apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.26.0),
+    tiledb (>= 0.26.0), tiledb (<= 0.26.99),
```

Comment:
Yep.  If and when tiledb-r changes version and if and when that would bring in a newer TileDB Embedded we'd have to lower the '99'.   But it is a 'known unknown' now and this, even just a signal value, may help.

Reply:
@eddelbuettel as always a bump to core 2.23 would be accompanied by a bump of TileDB-R to 0.27 -- if a TileDB-R 0.26.anything depends someday on a core 2.23, that would be an error.